js/index.js

```diff
@@ -0,0 +1,174 @@
+// toggle show-hide mobile menu
+const hamburger = document.querySelector('.hamburger-icon');
+const mobileNav = document.querySelector('.mobile-navigation');
+const defaultNav = document.querySelector('.navigation-menu');
+const xButton = document.querySelector('.img-x');
+
+function displayMobileMenu() {
+  defaultNav.style.display = 'none';
+  mobileNav.style.display = 'unset';
+}
+
+function hideMobileMenu() {
+  mobileNav.style.display = 'none';
+  defaultNav.style.display = 'unset';
+}
+
+hamburger.addEventListener('click', displayMobileMenu);
+xButton.addEventListener('click', hideMobileMenu);
+mobileNav.addEventListener('click', hideMobileMenu);
+
+// modal for project details
+const projects = [
+  {
+    number: 1,
+    title: 'Project name goes here',
+    techStack: ['HTML/CSS', 'Ruby on Rails', 'Javascript'],
+    imageUrlMobile: 'images/Rectangle_21.png',
+    imageUrlDesktop: '',
+    descriptions: [
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.',
+      'Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi Ut aliquip ex ea commodo consequat. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi.',
+    ],
+  },
```

Comment:
[OPTIONAL] Look to see how the structure can be improved 👀 one of the requirements was that each object contains at least the following information about the project: name, description, featured image, technologies, link to live version, and link to the project source code. Ie:
```
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```
Yours is lacking the link 🔗 you could try to add that 😉

Reply:
Ok, i will add it... Thank you so much